properties/P000065.md

```diff
@@ -1,10 +1,13 @@
 ---
 uid: P000065
-name: Continuum-sized
+name: Cardinality $=\mathfrak c$
+aliases:
+  - Continuum-sized
 refs:
   - mr: MR2048350
     name: General Topology (Willard)
 ---
-There is a bijection from $X$ to $2^\omega$.
+
+There exists a bijection from the space into $\mathbb R$.
```

Comment:
Replace "into" with "to" (otherwise it reads as a function from the space into R that is bijective between the space and the range of the function).  Or say "bijection between ... and ...".

Reply:
Addressed in afd393d